src/commands/device/register.ts

```diff
@@ -76,6 +76,6 @@ export default class DeviceRegisterCmd extends Command {
 			options.deviceType,
 		);
 
-		return result && result.uuid;
+		return result.uuid;
```

Comment:
not `result?.uuid`?

Reply:
Typing says uuid will be present (API confirmed the behavior) so this check was/should not be needed